pkg/upstream/admin-console.go

```diff
@@ -32,6 +32,29 @@ type UpstreamSettings struct {
 	IncludeMinio           bool
 }
 
+func WriteAdminConsoleFiles(renderDir string, options types.WriteOptions) error {
+	adminConsoleFiles, err := generateAdminConsoleFiles(renderDir, options)
+	if err != nil {
+		return errors.Wrap(err, "failed to generate admin console files")
+	}
+
+	for _, file := range adminConsoleFiles {
+		fileRenderPath := path.Join(renderDir, file.Path)
```

Comment:
same for `fileRenderPath`, we're not rendering anything, so maybe just `filePath`?

Reply:
see response above